packages/cyberstorm/src/components/componentStyles/TextInput.module.css

```diff
@@ -0,0 +1,68 @@
+/* Common textInput styles */
+.root {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  gap: var(--gap-m);
+  align-items: center;
+
+  padding: var(--padding-xs) var(--padding-m);
+  border: var(--size-6xs) solid var(--color-default-purple--3);
+  border-radius: var(--border-radius-3xs);
+  color: var(--color-text--tertiary);
+  background-color: var(--color-default-purple--3);
+}
+
+.root:hover {
+  border: var(--size-6xs) solid var(--color-default-purple--6);
+
+  --bg-color: var(--color-surface--4);
```

Comment:
Is this used anywhere?

Reply:
No it is not and I'm not sure why it is there 🤔 Removing it